lib/app.js

```diff
@@ -16,8 +17,18 @@ function app (opts) {
       throw Error('App not found (404)');
     }
 
-    return results[0];
-  });
+    const result = results[0];
+
+    if (opts.ratings) {
+      if (!opts.id) { opts.id = result.id; }
+      return Promise.all([result, ratings(opts)])
```

Comment:
I don't think you need the `Promise.all` here
```
      return ratings(opts)
          .then((ratingsResult) => Object.assign({}, result, ratingsResult));
```

Reply:
totally right, nice catch!

Reply:
added a commit